core/txpool/txpool.go

```diff
@@ -581,7 +581,7 @@ func (pool *TxPool) Locals() []common.Address {
 // account and sorted by nonce. The returned transaction set is a copy and can be
 // freely modified by calling code.
 func (pool *TxPool) local() map[common.Address]types.Transactions {
-	txs := make(map[common.Address]types.Transactions)
+	txs := make(map[common.Address]types.Transactions, len(pool.locals.accounts))
```

Comment:
This is not that obvious. There may be many local accounts but no transactions from them.

Reply:
> This is not that obvious. There may be many local accounts but no transactions from them.

Yes, it may be, for some public RPC nodes?